### PR TITLE
Fix crash on automatic import from git

### PR DESCRIPTION
**Pull request by martinvonz**
# Checklist

- [x] I have made relevant updates to `CHANGELOG.md`

### CODE REVIEW

File: tests/test_git_colocated.rs
Comment (spectral54) [inline tests/test_git_colocated.rs:58]: Is the bug actually fixed if we don't run this command?

**Review by spectral54 — APPROVED** [review]

File: tests/test_git_colocated.rs
Comment (martinvonz) [reply tests/test_git_colocated.rs:58]: Yes, the crash no longer happens, so the bug is fixed. What happens without this line is that a branch incorrectly becomes conflicted (same as you ran into earlier today, actually).

(Also, in case you hadn't realized, GitHub will merge as soon as you approve if auto-merge is on; comments won't prevent auto-merge. I wish it would have the author resolve any comments first, but that's not how it works.)

**Pull request merged**
